### ISSUE TITLE
Running both meteor packages tests and laika tests are conflicting

### ISSUE THREAD

**Issue by mitar** [arunoda/travis-ci-laika#1]
Running both meteor packages tests and laika tests are conflicting, because both download `Makefile`. Luckily, I can put calling `laika` directly into `.travis.yaml` as it is really simply to call, but it is something to think about. :-)

**Comment by arunoda** [comment]
Hmm. I never thought about running both at once. Lets see what we can do :)

**Comment by mitar** [comment]
I am not sure why you need Makefile in the first place. Running that one line could simply go into `.travis.yaml` `test` configuration.

**Comment by arunoda** [comment]
Yes. Good one. We may not need Makefile for laika. I used it on tinytest
because due to some odd npm issues. I'm not sure why exactly.

I just duplicate this for laika and never thought twice.

On Sun, Oct 20, 2013 at 1:25 PM, Mitar notifications@github.com wrote:

> I am not sure why you need Makefile in the first place. Running that one
> line could simply go into .travis.yaml test configuration.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/arunoda/travis-ci-laika/issues/1#issuecomment-26667653
> .

## 

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda
